### PR TITLE
fix: multiple instances

### PR DESCRIPTION
**Pull request by ml019**
*Description of changes:*
Place configuration within the scope of each instance rather than at the component level.

If multiple calls to the cfcognito module were made, link information for one call would override the information for the others. By moving under the instance, values will not override each other.

### CODE REVIEW

**Comment by roleyfoley** [comment]
As discussed this might be better with just the links under the instance. This allows for people to define their own instances in their solution from a single module instance and then just modify the link configuration on the new instances

File: hamlet/cfcognito/modules/cdnlambda/module.ftl [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
             "Names" : "tier",
-            "Description" : "The Id of this CDN instance",
+            "Description" : "The tier id of the CDN to create",
             "Type" : STRING_TYPE,
             "Mandatory" : true
         },
         {
             "Names" : "instance",
-            "Description" : "The id of the instance to create",
+            "Description" : "The instance id of the CDB to create",
```
Comment (roleyfoley) [inline hamlet/cfcognito/modules/cdnlambda/module.ftl]:
```suggestion
            "Description" : "The instance id of the CDN to create",
```

**Review by roleyfoley — CHANGES_REQUESTED** [review]
minor typo and comments around instances

**Comment by ml019** [comment]
I moved all the cdn config under instance as there were multiple references to the lambda CDN links. That won't work so well with the single module call and multiple instances in the solution, but with the data being injected into the lambdas such as the userpool info, the lambdas need to match the cdn I think. Thoughts?

The alternative is I go back and create two explicitly separate cdns,

**Comment by ml019** [comment]
Closing. Module changed to expect all instances of the primary cdn to share the same userpool client.  As a result, there is only one set of lambda functions per usage of the module.

If a cdn requires a different userpool client, then a different cdn component must be created so that a separate set of lambda (with different injected config) can be created.

**Pull request closed without merging**